drivers/src/memory_layout.rs

```diff
@@ -79,7 +79,7 @@ pub const DPE_SIZE: u32 = 5 * 1024;
 pub const PCR_RESET_COUNTER_SIZE: u32 = 1024;
 pub const AUTH_MAN_IMAGE_METADATA_MAX_SIZE: u32 = 7 * 1024;
 pub const IDEVID_CSR_SIZE: u32 = 1024;
-pub const DATA_SIZE: u32 = 20 * 1024;
+pub const DATA_SIZE: u32 = 148 * 1024;
```

Comment:
how is this sized?

Reply:
DATA_SIZE is the remaining space in DCCM after adding the other data structures in memory_layout.rs. Since DCCM is increasing by 128 KB, that increased amount goes to DATA_SIZE.